src/realm/table.cpp

```diff
@@ -3525,11 +3528,10 @@ void Table::typed_print(std::string prefix, ref_type ref) const
     std::cout << prefix << "}" << std::endl;
 }
 
-StringInterner* Table::get_string_interner(ColKey col_key) const
+StringInterner* Table::get_string_interner(ColKey::Idx idx) const
```

Comment:
This is for supporting collections in mixed right?

Reply:
No - it is to support a plain Mixed.